lib/src/auto_size_text_field.dart

```diff
@@ -718,6 +716,12 @@ class _AutoSizeTextFieldState extends State<AutoSizeTextField> {
     if (!widget.wrapWords) {
       var words = text.toPlainText().split(RegExp('\\s+'));
 
+      // Adds prefix and suffix text
+      if (widget.decoration.prefixText != null)
+        words.add(widget.decoration.prefixText);
+      if (widget.decoration.suffixText != null)
+        words.add(widget.decoration.suffixText);
```

Comment:
Hi @excogitatr This code change assumes that the `hintText` always shares the same textStyle as the input text which might not always be the case. We should consider the size of `hintText` with `hintStyle`. 

What do you think?

Reply:
Makes sense. Then should there be a seperate text painter and it will be added directly to `_textSpanWidth`?

Reply:
Absolutely. 👍

Reply:
BTW @lzhuor  I just realized that, prefix and suffix text is always same as the input text. It is completely different from hint. So, I don't think there will be any issue with this code.

Reply:
Gotcha. Good catch.